pkg/endpoint/endpoint.go

```diff
@@ -1797,3 +1802,71 @@ func (e *Endpoint) identityLabelsChanged(owner Owner, myChangeRev int) error {
 
 	return nil
 }
+
+// setPolicyRevision sets the policy wantedRev with the given revision.
+func (e *Endpoint) setPolicyRevision(rev uint64) {
```

Comment:
Could you leave a comment on Endpoint.policyRevision saying that you should modify it with this function only?